tests/js-unit/clipboard_test.ts

```diff
@@ -1,3 +1,4 @@
+import { assert, expect } from 'chai';
```

Comment:
These imports are known for free in the Cypress stack but with Karma/Mocha/Chai they need to be imported

Reply:
`assert` isn't used in this file - are we not linting the test scripts?

Reply:
Those folders were not linted before. Now covered (and more) with: https://github.com/plotly/dash-table/pull/878/commits/381e49de1d46e57bef864c221677b5e7f602e0ae, https://github.com/plotly/dash-table/pull/878/commits/c8985e37440e586c0a31b03fbd97c08aa8e23859,https://github.com/plotly/dash-table/pull/878/commits/6c086e393ff3fa9e0abff941b9c3618cfc40d7f1